JustFakeIt/ExpectationExtensions.cs

```diff
@@ -53,7 +56,48 @@ public static bool MatchesActualHttpMethod(this HttpRequestExpectation expected,
 
         public static bool MatchesActualBody(this HttpRequestExpectation expected, string actualBody)
         {
-            return string.IsNullOrEmpty(expected.Body) || actualBody.Equals(expected.Body);
+            return MatchBody(expected.Body, actualBody);
+        }
+
+        public static bool MatchBody(string expectedBody, string actualBody)
```

Comment:
Method name implies we're going ot be matching the entire body, instead it's only a partial match on expected key/value pairs.  
Suggest renaming to `BodyContains` or similar.